service/discovery/extra/csaf/document_properties.go

```diff
@@ -90,13 +90,16 @@ func clientAuthenticity(res *http.Response) *ontology.Authenticity {
 //
 // It uses the [documentChecksum] method to retrieve the respective checksum.
 func (d *csafDiscovery) documentChecksums(file csaf.AdvisoryFile, body []byte) (checksums []*ontology.DocumentChecksum) {
-	var toHash map[string]hash.Hash = map[string]hash.Hash{
-		constants.SHA_256: sha256.New(),
-		constants.SHA_512: sha512.New(),
+	var toHash = map[string]struct {
```

Comment:
Add test in test function with no error in response.

Reply:
This is hard to test, because we would need to invoke IO tests, etc. I would skip this for now